WWDC/WWDCLayer.swift

```diff
@@ -9,17 +9,13 @@
 import QuartzCore
 
 final class WWDCLayer: CALayer {
-
     override func action(forKey event: String) -> CAAction? {
         return nil
     }
-
 }
 
 final class WWDCShapeLayer: CAShapeLayer {
-
     override func action(forKey event: String) -> CAAction? {
         return nil
     }
-
```

Comment:
This whitespace removal is the only one I agree with, but I think other contributors like it.